src/core/ext/xds/xds_client_grpc.cc

```diff
@@ -115,11 +115,20 @@ const auto kMetricResourceUpdatesInvalid =
          kMetricLabelXdsResourceType},
         {}, false);
 
+const auto kMetricServerFailure =
+    GlobalInstrumentsRegistry::RegisterUInt64Counter(
+        "grpc.xds_client.server_failure",
+        "EXPERIMENTAL.  A counter of xDS servers going from healthy to "
```

Comment:
These description messages show up on the UI. I think we should try to be more concise.

Reply:
I would rather err on the side of verbosity.  There's nothing worse than trying to bebug a problem and not being entirely clear on what a metric actually indicates when you look at it.